.github/workflows/pypi_publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Upload Python Package to Test and Live Pypi repositories
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.6'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+
+    - name: Publish distribution to Test PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: __token__
+        password: ${{ secrets.test_pypi_password }}
+        repository_url: https://test.pypi.org/legacy/
+
+    - name: Publish distribution to PyPI
+      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
Isn't that the condition always true, when you listen to a "release-created"-event?

Reply:
Hmm I think you are right, removing it.